src/policy/policy.cpp

```diff
@@ -49,7 +49,9 @@ bool IsStandard(const CScript& scriptPubKey, txnouttype& whichType)
             return false;
         if (m < 1 || m > n)
             return false;
-    }
+    } else if (whichType == TX_NULL_DATA &&
+               (!GetBoolArg("-datacarrier", true) || scriptPubKey.size() > nMaxDatacarrierBytes))
```

Comment:
If `nMaxDatacarrierBytes` is 80, wouldn't the effective size only be 79 byte now, due to the op code, which is part of `scriptPubKey`?